package io.kyberorg.yalsee.exception;

import io.kyberorg.yalsee.ui.pages.err.ident404.IdentNotFoundPage;

/**
 * Exception, which not intended to be thrown, just {@link IdentNotFoundPage} handles it.
 *
 * @since 2.7.4
 */
public class IdentNotFoundException extends YalseeException {
}
